### PR TITLE
fix: fix attentionDP padding request type

### PR DESCRIPTION
**Pull request by Shunkangz**
The previous implement only targets IFB case where we can only fetch the context request from the request queue. In disaggregated serving, we can fetch both context request and generation request from it. In this case, we need to additional check condition to determine whether this request is context or not.

### CODE REVIEW

File: tensorrt_llm/_torch/pyexecutor/py_executor.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import torch
 
 from tensorrt_llm._utils import global_mpi_rank, nvtx_range
+from tensorrt_llm.bindings import executor as tllme
 from tensorrt_llm.bindings.executor import (FinishReason, InflightBatchingStats,
```
Comment (jinyangyuan-nvidia) [inline tensorrt_llm/_torch/pyexecutor/py_executor.py:21]: Since only `RequestType` is used in `tllme`, maybe we can just additionally import `RequestType` from `tensorrt_llm.bindings.executor`.

**Review by jinyangyuan-nvidia — APPROVED** [review]

File: tensorrt_llm/_torch/pyexecutor/py_executor.py (hunk 1 above)
Comment (Shunkangz) [reply tensorrt_llm/_torch/pyexecutor/py_executor.py:21]: Fixed. Thank you!
